### PR TITLE
feat: add ktor

### PR DESCRIPTION
**Pull request by edubacco**
As we have http4k in the radar, we should also add ktor.

### CODE REVIEW

File: radar/casavo-tech-radar.json [hunk 1]
```diff
@@ -530,5 +530,12 @@
     "quadrant": "tools",
     "isNew": "FALSE",
     "description": "We use it for asyncronous messaging"
+  },
+  {
+    "name": "ktor",
+    "ring": "adopt",
+    "quadrant": "languages & frameworks",
+    "isNew": "TRUE",
+    "description": "https://ktor.io/ asyncronous client and server kotlin http applications"
```
Comment (pikachuCa) [inline radar/casavo-tech-radar.json:539]: how much kotlin are we using ? is ktor used a lot at Casavo ?

Comment (xpepper) [reply radar/casavo-tech-radar.json:539]: We use Kotlin in several major sw components: Snake (backend of the valuation flow), Scotty (backend of the mobile app for data gathering and more), Gluglu (main backend of the listing platform), STS (a small tracking service).
We use ktor at least in Gluglu, don't know of other products.

**Comment by xpepper** [comment]
@edubacco I agree, I would also add a "why": what are the benefits of using Ktor?
(we could also copy the original TW's radar :D https://www.thoughtworks.com/radar/languages-and-frameworks/ktor)

File: radar/casavo-tech-radar.json (hunk 1 above)
Comment (edubacco) [reply radar/casavo-tech-radar.json:539]: channel managers are not anymore in kotlin with ktor, right?

**Review by pferretti — APPROVED** [review]

File: radar/casavo-tech-radar.json (hunk 1 above)
Comment (dottiOliviero) [reply radar/casavo-tech-radar.json]: channel managers are now developed in a single repo in python https://github.com/casavo/channel-managers

**Review by dottiOliviero — APPROVED** [review]

File: radar/casavo-tech-radar.json (hunk 1 above)
Comment (pikachuCa) [reply radar/casavo-tech-radar.json]: What is the language that we should use for a new backend service if the need arise ? 
Kotlin ? Scala ? Kotlin ?

Comment (pikachuCa) [reply radar/casavo-tech-radar.json]: The answer of my question should make us decide on what is the correct ring for ktor :)

**Review by moebiusmania — APPROVED** [review]

**Review by xpepper — APPROVED** [review]

**Pull request merged**